### ISSUE TITLE
/deep/ is deprecated by chrome and is being removed from angular support

### ISSUE THREAD

**Issue by lcecil** [Gbuomprisco/ngx-chips#537]
PLEASE MAKE SURE THAT:
- you searched similar issues online (9/10 issues in this repo are solved by googling, so please...do it)
- you provide an online demo I can see without having to replicate your environment
- you help me by debugging your issue, and if you can't, do go on filling out this form

**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x ] support request/question

Notice: feature requests will be ignored, submit a PR if you'd like
```

**Current behavior**

The custom themes use `/deep/` to apply the custom theme to the `<tag-input>` component. Since `/deep/` is deprecated as of Chrome 45, and Angular plans to remove this functionality, are there plans to adjust for this in the future?

See links below:
https://www.chromestatus.com/features/6750456638341120
https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep

**Expected behavior**

To be able to apply a theme that targets component classes without having to use a shadow-piercing descendant combinator.

**Minimal reproduction of the problem with instructions (if applicable)**

**What do you use to build your app?. Please specify the version**

angular-cli

**Angular version:** 

4.2.4

**ngx-chips version:**

1.5.2

**Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chrome 45 and up

**Comment by Gbuomprisco** [comment]
Hi @lcecil, 

Angular, as far as I know, is still working on a way to address this. Temporarily they have provided :ng-deep which you can use as replacement for /deep/. I will update the documentation and also change the references in the code

**Comment by lcecil** [comment]
Thanks!

I'll wait until the documentation is updated and I can check to make sure it works with my code before closing this issue.

**Comment by xtianus79** [comment]
it's ok as the replacement of ::ng-deep is still going to be used but is a little shaky in functinionality in all cases... ie. angular 2+ components

**Comment by LironHazan** [comment]
I still couldn't custom the tags when removed the /deep any news? 
Thanks! :)

**Comment by bytelabsco** [comment]
Even using ::ng-deep, it no longer seems possible to custom theme.  Is there a recommended work around?

**Comment by xtianus79** [comment]
@bytelabsco oddly... and perhaps google should be aware of this.  ng-deep is supposed to be the successor and /deep is supposed to be deprecated... but that is not the reality of the situation. /deep is something being deprecated on a browser sense... but it still works.  ng-deep taking over seemingly doesn't work and IMHO never has.  

I believe it is something that google needs to become aware of.

**Comment by bytelabsco** [comment]
After doing some more work yesterday, the problem I was having is 2 fold.  One is that ::ng-deep isn't working as expected.  The second is that a lot of the properties I would want to specify in a theme are simply not able to be set using the current theming variables.

For me personally, i'm not using material design and was looking for a chips component that could tie into bootstrap easily.  Using ngx-chips' bootstrap theme comes close, but I wanted to extend the .form-control definition from bootstrap in the ngx-chips container, but seems to be more trouble than it's worth.

**Comment by Gbuomprisco** [comment]
Changes addressed in v 1.7.4
